modelci/app/v1/endpoints/model.py

```diff
@@ -33,3 +33,33 @@ def get_all_model(name: str = None, framework: Framework_ = None, engine: Engine
 def get_model(*, id: str):  # noqa
     model = ModelService.get_model_by_id(id)
     return ModelDetailOut.from_bo(model)
+
+
+@router.get('/structure/{id}', response_model=...)
+async def get_model_structure(id: str):  # noqa
```

Comment:
Is this function defined with `async` keyword  because it may take a while to finish?

Reply:
Nope, this is to notify python that inside this function, you can use some async features.
This function is executed by FastAPI in the same way as a function without an async claim.
Therefore, under this senario, `async` does not have any effect.